content/ember-released-4-7.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Ember 4.7 Released
+authors:
+  - the-ember-learning-team
+date: 2022-09-5T00:00:00.000Z
+tags:
+  - releases
+  - '2022'
+  - version-4-x
+---
+
+Today the Ember project is releasing version 4.7 of Ember.js, Ember Data, and Ember CLI.
+
+This release kicks off the 4.7 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 4.7
+
+Ember.js 4.7 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Bug Fixes
+
+Ember.js 4.7 introduced 1 bug fix.
+
+- [Replacing Firefox detection that used a deprecated browser API](https://github.com/emberjs/ember.js/pull/20126)
+
+#### Features
+
+Ember.js 4.7 introduced 0 features.
+
+#### Deprecations
+
+Ember.js 4.7 introduced 0 deprecations.
+
+<!-- Block end -->
+
+For more details on changes in Ember.js 4.7, please review the [Ember.js 4.7.0 release page](https://github.com/emberjs/ember.js/releases/tag/v4.7.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 4.7
+
+Version 4.7 implements a "2.1" cache spec while deprecating the v1 cache spec. This spec differs somewhat from the [original 2.0 RFC](https://rfcs.emberjs.com/id/0461-ember-data-singleton-record-data), an updated RFC specifying the modifications will be published soon. Users looking to migrate to the 2.1 cache should expect that the finalized version of 2.1 enforces that the cache is always a singleton (vs merely allowed to be a singleton), and that data provided to the store and queries to the store from the cache should always contain identifiers in their stable form. This latter point is enforced in 4.7, the former is not yet enforced.
```

Comment:
The performance changes are the big callout to make, you can probably just directly copy that portion of the release notes:  https://github.com/emberjs/data/releases/tag/v4.7.1

Reply:
Cool, will add that too